packages/logging/package.json

```diff
@@ -56,7 +56,6 @@
     "eslint-plugin-jsx-a11y": "^2.2.2",
     "eslint-plugin-react": "^6.3.0",
     "jest": "20.0.3",
```

Comment:
Version is ko

Reply:
I have added an entry in the travis matrix to test logging and html plugin.